Auth/AuthConfig.ts

```diff
@@ -0,0 +1,35 @@
+import { Platform } from "react-native";
+import { AuthConfiguration } from "react-native-app-auth";
+import { AppEnvironment } from "../AppEnvironment";
+
+/**
+ * Generates a configuration to send OAuth servers for authentication or refresh.
+ * Relies on the current app environment to determine which URL to use.s
+ * @returns Current authentication config or null
+ */
+export default async function config(environment: AppEnvironment): Promise<AuthConfiguration|null> {
+
+    let response = await fetch(environment.baseUrl + '/api/v1/info');
+
+    if (! response.ok) {
+        console.log('no worky');
+        return null;
+    }
+
+    let json = await response.json();
+    console.log(json);
+
+    let config: AuthConfiguration = {
+        serviceConfiguration: {
+            authorizationEndpoint: environment.baseUrl + '/oauth/authorize',
+            tokenEndpoint: environment.baseUrl + '/oauth/token',
+        },
```

Comment:
I noticed the library supports setting an `issuer` instead, that might be a little cleaner. It will require a little bit of code on the server side but I can knock that out sometime this week.

Reply:
It's just the base URL for the server, e.g. https://apiary-sandbox.robojackets.org